irohad/consensus/yac/impl/yac_hash_provider_impl.cpp

```diff
@@ -16,22 +16,22 @@
  */
 
 #include "consensus/yac/impl/yac_hash_provider_impl.hpp"
+#include "interfaces/iroha_internal/block.hpp"
 
 namespace iroha {
   namespace consensus {
     namespace yac {
 
       YacHash YacHashProviderImpl::makeHash(
-          const shared_model::interface::BlockVariant &block_variant) const {
+          const shared_model::interface::Block &block) const {
         YacHash result;
-        auto hex_hash = block_variant.hash().hex();
+        auto hex_hash = block.hash().hex();
         result.proposal_hash = hex_hash;
         result.block_hash = hex_hash;
-        const auto &sig = *block_variant.signatures().begin();
+        const auto &sig = *block.signatures().begin();
         result.block_signature = clone(sig);
```

Comment:
I think, you can avoid using `sig` variable and pass its value into clone. It's pretty obvious, what `signatures().begin()` returns